go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	github.com/tchap/go-patricia/v2 v2.3.1
 	github.com/termie/go-shutil v0.0.0-20140729215957-bcacb06fecae
 	github.com/urfave/cli/v2 v2.27.5
-	github.com/vishvananda/netlink v1.2.1-beta.2.0.20240703200800-b54f85093f4a
+	github.com/vishvananda/netlink v1.3.0
```

Comment:
There is a [breaking change](https://github.com/vishvananda/netlink/pull/983/files#diff-d550ae620c1f4d7833c6b74d5c1898a6699529bf6f5a54688403426ca0e86ff2R14) in vishvananda/netlink that `Mask` is changed from `int` to `*uint32` (`Mark` from `int` to `uint32`). We may need to validate that in production code to avoid any nil pointer dereference or usage.